packages/sil-voting-app/src/components/Header.tsx

```diff
@@ -0,0 +1,33 @@
+import { Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+import Block from 'medulas-react-components/lib/components/Block';
+import CircleImage from 'medulas-react-components/lib/components/Image/CircleImage';
+import Typography from 'medulas-react-components/lib/components/Typography';
+import React from 'react';
+import icon from '../assets/iov-logo.svg';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  logo: {
+    '& > div + h5': {
+      marginLeft: theme.spacing(2),
+    },
+  },
+}));
+
+const Header = (): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Block width="100%" minHeight="78px" display="flex" alignItems="center">
+      <Block width="20%" display="flex" alignItems="center" justifyContent="center" className={classes.logo}>
```

Comment:
Strange way to do the layout. I am not saying it's incorrect, or bad code, just saying I am bit surprised. If parent's root Block uses "flex", then I expect its children to have automatic width regulated by `flexGrow`. 
If for any reason you need specific width, like in your case, I expected to use flexBasis property. Something like:
```
<Block width="100%" display="flex">
   <Block flexBasis="200">This is the left menu</Block>
   <Block flexGrow={1}>This is the main content</Block>
</Block>
```

Taken from css-tricks: 
> flex-basis -> This defines the default size of an element **before the remaining space is distributed**

So, you can assume flexGrow=1 will take all the space available.